.github/workflows/nightly_release.yml

```diff
@@ -47,15 +50,23 @@ jobs:
           # Install node dependencies
           node ./scripts/gtime.js monorepo_install npm ci
 
+      - name: Update identity in git config
+        run: |-
+          git config --global user.name ${{ secrets.GIT_CONFIG_USERNAME }}
+          git config --global user.email ${{ secrets.GIT_CONFIG_EMAIL }}
+
       - name: Create Release Branch
-        run: git switch release-nightly-${{ env.monorepo_version_major }}.x || git switch -c release-nightly-${{ env.monorepo_version_major }}.x
+        run: |-
+          git fetch
+          git switch develop
+          git switch nightly-releases || git switch -c nightly-releases
 
         # Pull all changes from develop and keep changes from develop in case of merge-conflicts
       - name: Pull latest changes from develop
         run: |-
           git switch develop
-          git merge -s ours release-nightly-${{ env.monorepo_version_major }}.x
-          git switch release-nightly-${{ env.monorepo_version_major }}.x
+          git merge -s ours nightly-releases
```

Comment:
Totally missed that `release-*` branches have branch protection rules and will not allow pushing code without a PR so had to switch branch name to `nightly-releases`